extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -33,9 +34,23 @@ let PropertyRulesDef;
  * @private {Object<string, ?PropertyRulesDef>}
  */
 const GLOBAL_PROPERTY_RULES = map({
+  'text': null,
```

Comment:
Do we want to allow any tag to have the `[text]` attribute? I could see this being potentially problematic on a `<div> `for instance, where reassigning its textContent could be used to remove its contents.